README.md

```diff
@@ -1,5 +1,38 @@
-# pWallet
+# Persistence Wallet
+This is the main repository Persistence Wallet, React + Javascript project.
 
-Requires Rest Client CORS and RPC client CORS.
+## Deployments
+Tagged Releases are deployed on [wallet.persistence.one](https://wallet.persistence.one) which connects to Peristence Core mainnet.   
+Latest main branch codebase is deployed on [staging.wallet.persistence.one](https://staging.wallet.persistence.one) which connects to Persistence Core testnet.
+Any requests for test-tokens, questions, suggestions, are welcome on [Discord Persistence -> under wallet-development](https://discord.gg/s8hBStXjKs)
 
-please take a look at config before release
+## Libraries and Integrations   
+- [cosmos/cosmjs](https://github.com/cosmos/cosmjs)
+- [cosmostation/cosmosjs](https://github.com/cosmostation/cosmosjs)
+- [keplr experimental feature](https://github.com/chainapsis/keplr-example)
+## Developer Configurations
+```
+npm: '7.14.0'
+node: '16.1.0'
+```
+
+.env 
+```
+# REACT_APP_API_KEY is the blockchain rest endpoint, default ~ ip:1317
+REACT_APP_API_KEY =https://persistence.testnet.rest.audit.one
+# REACT_APP_TENDERMINT_RPC_ENDPOINT is the tendermint rpc endoiunt, default ~ ip:26657
+REACT_APP_TENDERMINT_RPC_ENDPOINT=https://persistence.testnet.rpc.audit.one
+# REACT_APP_MAIL_CHIMP_URL is required for mail subscription ~ not required for local development
+REACT_APP_MAIL_CHIMP_URL=
```

Comment:
Since this value needs to be set, maybe move it either to the TOP or the bottom, with extra space between default values and values that need to be added

Reply:
empty is fine, got it